### ISSUE TITLE
Temp hack fix for Opencalais

### ISSUE THREAD

**Issue by fingertrouble** [TaxoPress/taxopress#137]
Sorry have not used Git for pull requests before - I have added one here: https://github.com/BeAPI/simple-tags/pull/136 - but for those with live servers here's how to fix it - I posted this over at the Simple-Tags WP Support forum but it got moderated - I guess the links - so who knows if that ever will see the light of day.

I knew that the Opencalais tagging still worked cos my old key worked in Laiser Tag, a newer and working auto-tag WP plugin that only does Opencalais, problem is unlike Simple Tags there isn’t any way of selecting the tags manually.

So this is what you do 

1. Find /simple-tags in your WordPress plugin folder – usually in /wp-content/plugins, then go into the /inc/ folder

2. Edit the class.admin.suggest.php file in a text-editor and search for this line:

`$response = wp_remote_post( 'https://api.thomsonreuters.com/permid/calais'`

Change it to:

`$response = wp_remote_post( 'https://api-eit.refinitiv.com/permid/calais'`

Basically the new API URL is: https://api-eit.refinitiv.com/permid/calais and needs to be pointed to that URL. Why Refinitiv/Thomson Reuters couldn’t redirect it, I don’t know.

Remember this hack will be overwritten if the plugin is updated or you re-install/re-upload it! Unless this pull request is successful, that is.

**Comment by stevejburge** [comment]
Thanks @fingertrouble. We're looking at this fix
